packages/salesforcedx-vscode-lwc/src/index.ts

```diff
@@ -14,11 +14,14 @@ import {
   ServerOptions,
   TransportKind
 } from 'vscode-languageclient';
-import { forceLightningLwcCreate } from './commands/forceLightningLwcCreate';
 import { ESLINT_NODEPATH_CONFIG, LWC_EXTENSION_NAME } from './constants';
 import { nls } from './messages';
 
 function registerCommands(): vscode.Disposable {
+  const {
```

Comment:
This needs to be done using a dynamic import (basically a require). 
The reason being that if you use an import, it is processed the moment the file is loaded through the Node.js module system. We don't really want that. We only want the ./commands/forceLightningLwcCreate file to be processed when we have successfully activated the rest of its dependencies (vscode dependencies, not Node dependencies).

This doesn't have any impact in real usage but for the tests where things are more controlled, this guarantees that our salesforce.salesforcedx-vscode-core is loaded first.